newPost.sh

```diff
@@ -0,0 +1,46 @@
+read -p "Title: " title
```

Comment:
Add a shebang at the top of the file. The [Wikipedia Page](https://en.wikipedia.org/wiki/Shebang_(Unix)) for Shebang is actually quite useful and informative for information on what it is and how it works.